docs/resources/client/models/bakedmodel.md

```diff
@@ -0,0 +1,117 @@
+
+- An `ItemDisplayContext`: The [perspective] the model is being transformed to.
+- A `PoseStack`: The pose stack used for rendering.
+- A `boolean`: Whether to use modified values for left-hand rendering instead of the default right hand rendering; `true` if the rendered hand is the left hand (off hand, or main hand if left hand mode is enabled in the options)
+
+### Others
+
+Other methods in `BakedModel` that you may override and/or query include:
+
+| Signature                                                                     | Effect                                                                                                                                                                                                                             |
+|-------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `boolean useAmbientOcclusion()`                                               | Whether to use [ambient occlusion][ao] or not. Has an override that accepts a `BlockState` parameter, and an override that accepts a `BlockState` and a `RenderType` parameter.                                                    |
+| `boolean isGui3d()`                                                           | Whether to use a 3d renderer for rendering in GUIs or not. Also affects GUI lighting.                                                                                                                                              |
+| `boolean usesBlockLight()`                                                    | Whether to use block light when lighting the model or not.                                                                                                                                                                         |
+| `boolean isCustomRenderer()`                                                  | If true, skips normal rendering and calls an associated [block entity renderer][ber]'s `renderByItem` method instead. If false, renders through the default renderer.                                                              |
+| `ItemOverrides getOverrides()`                                                | Returns the [`ItemOverrides`][itemoverrides] associated with this model. This is generally only relevant on item models.                                                                                                           |
+| `ModelData getModelData(BlockAndTintGetter, BlockPos, BlockState, ModelData)` | Returns the model data to use for the model. Mainly used for [block entity renderers][ber].                                                                                                                                        |
```

Comment:
Actually `getModelData` is probably used more often by models for blocks without a block entity (e.g. connected texture mods, since they can figure out what blocks the current block is surrounded by). Block entities already have a way of supplying model data via `BlockEntity#getModelData`.